package.json

```diff
@@ -32,6 +32,7 @@
     "@rollup/plugin-commonjs": "^15.0.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^9.0.0",
+    "@rollup/plugin-typescript": "^6.0.0",
```

Comment:
was this change necessary for this fix?

Reply:
Yes, it fixed errors with the exports being incorrectly formatted since rollup was working off of the TS output it was translating from commonjs back to esm and this just made it easier to reason about the pipeline.